packages/phoenix-evals/src/phoenix/evals/templates.py

```diff
@@ -240,6 +254,27 @@ def normalize_prompt_template(template: Union[PromptTemplate, str]) -> PromptTem
     )
 
 
+def patched_explanation_template(
+    template_parts: List[PromptPartTemplate],
+) -> List[PromptPartTemplate]:
+    """
+    Attempts to patch a template to additionally include an explanation part.
+    """
+    patched_explanation_template_text = (
+        "*****\n\n"
+        "After following the previous instructions, add a paragraph that starts with "
+        "`EXPLANATION: ` and then provide a concise explanation of your reasoning."
+    )
```

Comment:
how do we know that the original template doesn't contain this instruction? I think there might be downsides to the fact that now the template is not declarative but constructed inside of the function itself?

The scenario I'm worried about is that Arize is now trying to build templates that are "evaluation" templates. Depending on how you pass it into this, the explanation postfix will now get added (possibly twice).

We should probably be careful here so I'm gonna block for now just to make sure we don't break any workflows.